.vscode/settings.json

```diff
@@ -69,5 +69,8 @@
     "wsol",
     "WSOLATA"
   ],
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "rust-analyzer.linkedProjects": [
+    "./programs/uxd/Cargo.toml"
+  ]
```

Comment:
interdasting, im curious what this is for

Reply:
Me too, vscode what have you done 😄